src/base/static/components/molecules/add-place-button.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "react-emotion";
+import { connect } from "react-redux";
+
+import { Button } from "../atoms/buttons";
+import mq from "../../../../media-queries";
+
+import {
+  addPlaceButtonVisibilitySelector,
+  geocodeAddressBarVisibilitySelector,
+} from "../../state/ducks/ui";
+
+const AddPlaceButton = styled(
+  props =>
+    props.isAddPlaceButtonVisible && (
+      <Button
+        size="extra-large"
+        variant="raised"
+        color="primary"
+        className={props.className}
+        onClick={props.onClick}
+      >
+        {props.children}
+      </Button>
+    ),
+)(props => {
+  return {
+    zIndex: 10,
+    backgroundColor: props.theme.map.addPlaceButtonBackgroundColor,
+
+    "&:hover": {
+      backgroundColor: props.theme.map.addPlaceButtonHoverBackgroundColor,
+    },
+
+    [mq[0]]: {
+      position: "fixed",
+      width: "100%",
+      bottom: 0,
+      borderRadius: 0,
+    },
+    [mq[1]]: {
+      position: "absolute",
+      top: props.isGeocodeAddressBarVisible ? "92px" : "20px",
```

Comment:
Some magic numbers here, but I think we should punt this for later. We should think through how we want to handle positioning of the major pieces of UI.